lib/models/BucketInfo.ts

```diff
@@ -188,7 +188,7 @@ export default class BucketInfo {
         notificationConfiguration?: any,
         tags?: Array<BucketTag> | [],
         capabilities?: Capabilities,
-        quotaMax?: number | 0,
+        quotaMax?: bigint,
```

Comment:
should we not be more leniant, to avoid issues: e.g. allow the api to take `number | bigint | 0`, and initalize the field with `this._quotaMax = BigInt(quotaMax || 0n)` ?
or is there a performance benefit to doing this?

Reply:
The API accepts strings only since the beginning. Here, this is just how we parse/handle the value internally, it must be bigint here because we control how we create the BucketInfo class, no need to keep debt here.